docs/conf.py

```diff
@@ -57,7 +58,10 @@ def __repr__(self):
 
 project = "tpm2-pytss"
 author = "tpm2-software"
-copyright = f"2019 - {datetime.datetime.today().year}, {author}"
+build_date = datetime.datetime.utcfromtimestamp(
+    int(os.environ.get('SOURCE_DATE_EPOCH', time.time()))
```

Comment:
Style checker failed you here, you need to use `"` (double quotes) over `'` (single quotes).

```
+ python3 -m black --diff --check /home/runner/work/tpm2-pytss/tpm2-pytss
--- /home/runner/work/tpm2-pytss/tpm2-pytss/docs/conf.py	2022-10-26 15:02:[16](https://github.com/tpm2-software/tpm2-pytss/actions/runs/3323856520/jobs/5508219874#step:4:17).448387 +0000
+++ /home/runner/work/tpm2-pytss/tpm2-pytss/docs/conf.py	2022-10-26 15:02:22.543574 +0000
@@ -57,11 +57,11 @@
 # -- Project information -----------------------------------------------------
 
 project = "tpm2-pytss"
 author = "tpm2-software"
 build_date = datetime.datetime.utcfromtimestamp(
-    int(os.environ.get('SOURCE_DATE_EPOCH', time.time()))
+    int(os.environ.get("SOURCE_DATE_EPOCH", time.time()))
 )
 copyright = f"20[19](https://github.com/tpm2-software/tpm2-pytss/actions/runs/3323856520/jobs/5508219874#step:4:20) - {build_date.year}, {author}"
 
 # The short X.Y version
 version = get_version(root="..", relative_to=__file__)
would reformat /home/runner/work/tpm2-pytss/tpm2-pytss/docs/conf.py
Oh no! 💥 💔 💥
1 file would be reformatted, 35 files would be left unchanged.
Error: Process completed with exit code 1.
```

Reply:
Thanks for decoding this: force-pushed. :)